MCprep_addon/materials/default_materials.py

```diff
@@ -126,20 +126,19 @@ def execute(self, context):
             return {'CANCELLED'}
         
         if sync_file == bpy.data.filepath:
-			
             return {'CANCELLED'}
 
         # ------------------------- Find the default material ------------------------ #
-        MaterialName = f"default_{self.SIMPLE if not self.UsePBR else self.PBR}_{self.Engine}"
-        if not default_material_in_sync_library(MaterialName, context):
+        material_name = f"default_{self.SIMPLE if not self.UsePBR else self.PBR}_{self.engine}"
```

Comment:
I didn't catch this before: f-strings aren't available in older python versions (ie before blender's python in 2.80), plus this comprehension is rather long anyways. I'd use an old-style string formatter:

```
material_name = "default_{form}_{engine}".format(
    form=self.SIMPLE if not self.UsePBR else self.PBR,
    engine=self.engine
)
```